docs/how-to/hip_runtime_api/opengl_interop.rst

```diff
@@ -0,0 +1,93 @@
+.. meta::
+   :description: HIP provides an OpenGL interoperability API that allows
+                 efficient data sharing between HIP's computing power and
+                 OpenGL's graphics rendering.
+   :keywords: AMD, ROCm, HIP, OpenGL, interop, interoperability
+
+*******************************************************************************
+OpenGL interoperability
+*******************************************************************************
+
+The HIP--OpenGL interoperation involves mapping OpenGL resources, such as
+buffers and textures, for HIP access. This mapping process enables HIP to
+utilize these resources directly, bypassing the need for costly data transfers
+between the CPU and GPU. This capability is useful in applications that require
+both intensive GPU computation and real-time visualization.
+
+The graphics resources must be registered using functions like
+:cpp:func:`hipGraphicsGLRegisterBuffer` or :cpp:func:`hipGraphicsGLRegisterImage`
+then they can be mapped to HIP with :cpp:func:`hipGraphicsMapResources`
+function.
+
+After mapping, the :cpp:func:`hipGraphicsResourceGetMappedPointer` or
+:cpp:func:`hipGraphicsSubResourceGetMappedArray` functions used to retrieve a
+device pointer to the mapped resource, which can then be used in HIP kernels.
+
+Unmapping resources with :cpp:func:`hipGraphicsUnmapResources` after
+computations ensure proper resource management.
+
+Example
+===============================================================================
+
+ROCm examples have a `HIP--OpenGL interoperation example <https://github.com/ROCm/rocm-examples/tree/develop/HIP-Basic/opengl_interop>`_,
+where a simple HIP kernel is used to simulate a sine wave and rendered to a
+window as a grid of triangles using OpenGL. For a working example, there are
+multiple initialization steps needed like creating and opening a window, 
+initializing OpenGL or selecting the OpenGL-capable device. After the
+initialization in the example, the kernel simulates the sinewave and updates
+the window's framebuffer in a cycle until the window is not closed.
```

Comment:
until the windows is closed.

Reply:
There is only one window in the example, and it's updated in a while loop until the window is not closed.